### PR TITLE
Document usage of ProductOptions with a Shipment

### PR DESCRIPTION
**Pull request by basepack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,19 @@ $shipmentResponse = $client->shipments()->create(
 $label = $shipmentResponse->getLabel();
 ```
 
+### Adding ProductOptions to a Shipment
+```
+$shipment = new Shipment();
+$productOptions = [
+    ( new ProductOption() )->setOption( ProductOption::OPTION_ALLOW_NEIGHBOURS )->setValue( true ),
```
Comment (jacobdekeizer) [inline README.md]: Could you remove the white spaces between ( ), so it looks like this:
`(new ProductOption())->setOption(ProductOption::OPTION_ALLOW_NEIGHBOURS)->setValue(true),`

**Comment by jacobdekeizer** [comment]
Hi, thank you for your contribution. I've left some minor comments to the code. Could you take a look at them. If that is fixed, it is ready to be merged!

**Comment by basepack** [comment]
Good point, see the updated patch-3 branch.

**Comment by jacobdekeizer** [comment]
@basepack awesome, thank you

**Comment by jacobdekeizer** [comment]
@basepack Version `1.1.0` is now available.

**Comment by basepack** [comment]
@jacobdekeizer thanks for merging that fast, updating now...

**Pull request merged**